### PR TITLE
core: remove aws quicksight

### PR DESCRIPTION
**Pull request by BrianJayChung**
removed AWS QuickSight and all of its dependencies.

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -121,13 +121,6 @@ def dataexport():
     session['ld_user'] = user
 
     # if AWS creds are not set, don't show the dashboard
```
Comment (levlaz) [inline app/routes.py]: Can you remove the comments please?

File: app/routes.py [hunk 2]
```diff
@@ -121,13 +121,6 @@ def dataexport():
...
-    try:
-
-        embed_url = fetch_aws_embed_url()
-
-    except botocore.exceptions.NoCredentialsError as e:
-
-        current_app.logger.debug(e)
 
     show_data_export = ldclient.get().variation(
```
Comment (levlaz) [inline app/routes.py]: Can we get rid of this flag since we're removing the feature?

File: app/config.py [hunk 3]
```diff
@@ -45,12 +45,8 @@ class Config(object):
     REDIS_URL = os.environ.get('REDIS_URL')
     CACHE_CONFIG = {'CACHE_TYPE': 'simple'}
 
-    AWS_QUICKSIGHT_ACCESS_KEY_ID = os.environ.get('AWS_QUICKSIGHT_ACCESS_KEY_ID')
-    AWS_QUICKSIGHT_SECRET_ACCESS_KEY_ID = os.environ.get('AWS_QUICKSIGHT_SECRET_ACCESS_KEY_ID')
-    AWS_ACCOUNT_ID = os.environ.get('AWS_ACCOUNT_ID')
-    AWS_QUICKSIGHT_DASHBOARD_ID = os.environ.get('AWS_QUICKSIGHT_DASHBOARD_ID')
-    AWS_QUICKSIGHT_SESSION_LIFE = 100
-    AWS_QUICKSIGHT_REGION = "us-west-2"
+    # for future use
```
Comment (levlaz) [inline app/config.py]: Can we get rid of this?

Comment (BrianJayChung) [reply app/config.py]: Want me get rid of this too?

>AWS_ACCOUNT_ID = os.environ.get('AWS_ACCOUNT_ID')

**Comment by levlaz** [comment]
@BrianJayChung FYI docker was having issues yesterday which is why your tests were failing.

File: app/routes.py [hunk 4]
```diff
@@ -120,27 +120,12 @@ def dataexport():
...
-        False)
-
     set_theme = '{0}/dataexport.html'.format(current_user.set_path)
 
     return render_template (
         set_theme,
         title='dataexport',
-        embed_url=embed_url,
-        show_data_export=show_data_export
+        embed_url=embed_url
```
Comment (levlaz) [inline app/routes.py]: @BrianJayChung I think you meant to get rid of this right?

**Comment by BrianJayChung** [comment]
Thanks for reviewing this @levlaz , dump.rdp file should be removed with now.

**Comment by levlaz** [comment]
@BrianJayChung excellent, thank you!

**Pull request merged**